packages/tpetra/core/cmake/TpetraCore_config.h.in

```diff
@@ -174,6 +174,9 @@
 #  define TPETRACORE_F77_BLAS_MANGLE@F77_BLAS_MANGLE@
 #endif
 
+
+#cmakedefine HAVE_TPETRA_EXPERIMENTAL
```

Comment:
The CMake option gets defined in `tpetra/CMakeLists.txt`, but the macro gets defined in TpetraCore.  There's plenty of precedent for that, but at some point (doesn't have to be this PR), we should add a `tpetra/cmake/Tpetra_config.h` header file for Tpetra-wide macros like this.

Reply:
Yeah, we should be more consistent about this.  Add it to the Tpetra team backlog :)